src/backend/distributed/operations/stage_protocol.c

```diff
@@ -390,7 +390,6 @@ InsertShardPlacementRows(Oid relationId, int64 shardId, List *workerNodeList,
 						 int workerStartIndex, int replicationFactor)
 {
 	int workerNodeCount = list_length(workerNodeList);
-	int placementsInserted = 0;
 	List *insertedShardPlacements = NIL;
 
 	for (int attemptNumber = 0; attemptNumber < replicationFactor; attemptNumber++)
```

Comment:
Maybe rename attemptNumber to placementIndex.

I'm guessing there was some failover logic here in the past, and "attempt" is no longer accurate.